src/main/java/com/aws/greengrass/deployment/DefaultDeploymentTask.java

```diff
@@ -111,6 +114,20 @@ public DeploymentResult call() throws InterruptedException {
                     .kv("Deployment service config", deploymentServiceConfig.toPOJO().toString())
                     .log("Starting deployment task");
 
+            // Enforce deployment sequence validation using timestamps:
+            // If we have already processed a more recent deployment for a thing group, then we will drop
+            // this deployment and only process it if it is newer (or same) deployment for that thing group.
+            if (isStaleDeployment(deploymentDocument)) {
+                logger.atInfo().setEventType(DEPLOYMENT_TASK_EVENT_TYPE)
+                        .kv("Deployment service config", deploymentServiceConfig.toPOJO().toString())
```

Comment:
I don't think we should be dumping the entire config, particularly at info level.

Reply:
I am removing that, as the config is already dumped right above this check.